packages/components/tooltip/src/tooltip.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import type { ComponentProps } from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import Constraints from '@commercetools-uikit/constraints';
+import { PrimaryButton } from '@commercetools-uikit/buttons';
+import type { Meta, StoryObj } from '@storybook/react';
+import Tooltip from './tooltip';
+
+const CustomWrapper = styled.div`
+  display: block;
+  background-color: pink;
+`;
+
+const CustomBody = styled.div`
+  font-size: 0.875rem;
+  color: red;
+`;
+
+type TooltipPropsAndCustomArgs = ComponentProps<typeof Tooltip> & {
+  fullWidth?: boolean;
+  customBodyWrapper?: boolean;
+};
+
+const meta = {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  tags: ['autodocs'],
+} satisfies Meta<TooltipPropsAndCustomArgs>;
+
+export default meta;
+
+type Story = StoryObj<TooltipPropsAndCustomArgs>;
+
+export const Default: Story = {
+  args: {
+    off: false,
+    title: 'Tooltip text.',
+    showAfter: 300,
+    closeAfter: 200,
+    children: (
+      <div
+        css={css`
+          width: min-content;
+          cursor: not-allowed;
+          > :disabled {
+            pointer-events: none;
+          }
+        `}
+      >
+        <PrimaryButton onClick={() => {}} label="Submit" />
+      </div>
+    ),
+    fullWidth: false,
+    customBodyWrapper: false,
+  },
+  argTypes: {
+    placement: {
+      control: 'select',
+      options: [
+        'top',
+        'top-start',
+        'top-end',
+        'right',
+        'right-start',
+        'right-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'left',
+        'left-start',
+        'left-end',
+      ],
+      defaultValue: 'top',
+    },
+    horizontalConstraint: {
+      control: 'select',
+      options: Constraints.getAcceptedMaxPropValues(),
+      defaultValue: 'scale',
+    },
```

Comment:
I know this is a small definition but I think we will be using it in many other components.
What do you think about creating a shared helper similar to what we have for icons?